composeCore/src/main/java/com/hamdan/forzenbook/compose/core/theme/Dimens.kt

```diff
@@ -23,6 +25,8 @@ class Dimens(
     buttonPadVertical_1: Dp,
     buttonPadVertical_2: Dp,
 ) {
+    var borderStrokeSmall by mutableStateOf(stroke1, structuralEqualityPolicy())
```

Comment:
I've seen examples of people making var states for things like this, but it doesn't make sense to me at all

Reply:
Look at how Pinnacle handles Dimens for example of what I mean. It can just be simple

Reply:
I think I want to do a larger dimens refactor and apply the grid idea to it. So if I can I'd like to do that in a PR branced off this one.